test/permissibleRender.test.js

```diff
@@ -152,12 +170,13 @@ describe('PermissibleRender', () => {
     searchedElement.length.should.be.greaterThan(0);
   });
 
-  it('doesn\'t render a component if the user hasn\'t all of necessary permissions when `oneperm` prop isn\'t defined', () => {
-    props = {
+  it('doesn\'t render a component if the user doesn\'t have all of necessary permissions when `oneperm` prop is explicitely set to false', () => {
```

Comment:
Typo. Please use `explicitly` instead of `explicitely`

Reply:
Fixed.